torch/csrc/api/include/torch/enum.h

```diff
@@ -68,6 +73,11 @@ struct enum_name {
   TORCH_ENUM_PRETTY_PRINT(Reflect)
   TORCH_ENUM_PRETTY_PRINT(Replicate)
   TORCH_ENUM_PRETTY_PRINT(Circular)
+  TORCH_ENUM_PRETTY_PRINT(Nearest)
+  TORCH_ENUM_PRETTY_PRINT(Bilinear)
+  TORCH_ENUM_PRETTY_PRINT(Bicubic)
+  TORCH_ENUM_PRETTY_PRINT(Trilinear)
+  TORCH_ENUM_PRETTY_PRINT(Area)
```

Comment:
Thanks so much for add the variant-based enums! I think we could add tests for the new enums to `test/cpp/api/enum.cpp` as well, to make sure that people can use the new enums by simply including `torch/torch.h` :D